docs/requirements.txt

```diff
@@ -0,0 +1,27 @@
+alabaster==0.7.12
+Babel==2.8.0
+certifi==2019.11.28
+chardet==3.0.4
+docutils==0.16
+idna==2.8
+imagesize==1.2.0
+Jinja2==2.10.3
+m2r==0.2.1
```

Comment:
This is the package I have used for conversion

Reply:
Nice, works just fine now!

I need to allocate some time to merge this and set up RTD for bender. Since it's https://www.devconf.info/cz/ happening this week, it will take me some time to do that. Mehant, unless you want set up RTD yourself :)